java/src/plume/WeakHasherMap.java

```diff
@@ -117,7 +118,9 @@
  * @see		java.lang.ref.WeakReference
  */
 
-@SuppressWarnings({"nullness", "keyfor", "interning", "purity"}) // old, non-typesafe Sun code, not worth annotating or checking
+@SuppressWarnings({"nullness", "keyfor", "interning", "purity"}) // old, non-typesafe Sun code, not worth annotating or checking.
+// Annotated for lock checking as an experiment in annotating non-typesafe code.
```

Comment:
I suggest that the annotations be removed from this file.  That will also make it easier to compare this file with its upstream version.